components/automate-chef-io/content/docs/datafeed.md

```diff
@@ -69,9 +69,10 @@ To delete a Data Feed instance in Chef Automate:
 The settings in `config.toml` apply across all configured Data Feed instances.
```

Comment:
@satellite15 Do we need to specify which `config.toml` we're referring to here since we're erasing contextual information elsewhere?

Reply:
Can be changed to

The data-feed-service configuration applies across all configured Data Feed instances.